source/includes/azure/_public_ip_addresses.md

```diff
@@ -143,3 +143,35 @@ Optional | &nbsp;
 `allocationMethod` <br/>*string* | Allocation method of the public IP address. Possible values: `DYNAMIC`, `STATIC`. Default value is `DYNAMIC` for SKU `BASIC`, and `STATIC` for SKU `STANDARD`.
 `idleTimeout` <br/>*integer* | The number of minutes for the idleTimeout. I can be between 4 and 30 minutes. Default value is 4 minutes.
 `domainName` <br/>*string* | The subdomain part of the fqdn.
+
+
+<!-------------------- UPDATE A PUBLIC IP -------------------->
+
+#### Update a public ip address
+
+```shell
+curl -X PUT \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body"
+   "https://cloudmc_endpoint/v1/services/azure/example/publicipaddresses//subscriptions/subscriptionId/resourceGroups/cmc-example/providers/Microsoft.Network/publicIPAddresses/some-public-ip"
+
+# Request Example:
+```
+
+```json
+{
+	"allocationMethod" : "DYNAMIC",
+	"idleTimeout" : 30,
+	"domainName" : "samplePublicIP"
+}
+```
+
+<code>PUT /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/publicipaddresses/:id</code>
+
+Update a public IP address in a given [environment](#administration-environments)
+
+Attribute | &nbsp;
+------- | -----------
+`allocationMethod` <br/>*string* | Allocation method of the public IP address. Possible values: `DYNAMIC`, `STATIC`. Not providing it will keep the actual value. Cannot be change for Standard SKU public IP address.
+`idleTimeout` <br/>*integer* | The number of minutes for the idleTimeout. I can be between 4 and 30 minutes. Not providing it will keep the actual value.
```

Comment:
```suggestion
`idleTimeout` <br/>*integer* | The number of minutes for the idleTimeout. It can be between 4 and 30 minutes. Not providing it will keep the actual value.
```
Also same comment about 'actual'

Reply:
i means it does not change. The actual value in azure.

Reply:
Will fix both place for the i -> it

Reply:
60cbdca059792992ecae9c7f56f6741d2e7778e4